libpacaur/deps.sh

```diff
@@ -263,6 +274,11 @@ FindDepsAur() {
     fi
 }
 
+##
+# Sort dependencies to ensure correct resolution order.
+#
+# usage: SortDepsAur( $aur_packages )
```

Comment:
Note #652 here. This would ideally be merged with FindDepsAur() - I just haven't took time to do it properly yet.